test/e2e-test/ci-e2e-test.sh

```diff
@@ -5,9 +5,26 @@ This is an E2E test script.
 If you want to run the script , please run the ci-setup.sh script in advance to set up the ci test environment.
 '
 
-WORK_DIR=$(cd "$(dirname "$0")"; pwd)
+PY_WORK_DIR='test/e2e-test/py-test'
```

Comment:
Is it better to split the action into python_test and go_test? Since the github action result will be used as test report, it's clear to see both python_test and go_test passed.